### ISSUE TITLE
throw descriptive error when using a style without a valid `base`

### ISSUE THREAD

**Issue by meetar** [tangrams/tangram#618]
**TANGRAM VERSION:**

v0.14

**ENVIRONMENT:**

Mac OS X Chrome

**TO REPRODUCE THE ISSUE, FOLLOW THESE STEPS:**

Test scene includes an `outline` with a `style` specified without a `base`:

v0.13:
https://mapzen.com/tangram/play/?api=5/1188&lib=https://mapzen.com/tangram/0.13/tangram.debug.js

v0.14:
https://mapzen.com/tangram/play/?api=5/1188&lib=https://mapzen.com/tangram/0.14/tangram.debug.js

**RESULT:**

When using v0.14, drawing fails with:

```js
Tangram v0.14.1 [error]: Error building tile mapzen/12/1205/1541/12: TypeError: Cannot read property 'dash' of undefined
    at Object._preprocess (blob:https://mapzen.com/81d4d985-0c44-4a18-bf12-6c4b4dd762e0:36919:101)
    at Object.preprocess (blob:https://mapzen.com/81d4d985-0c44-4a18-bf12-6c4b4dd762e0:38636:25)
    at Function.buildGeometry (blob:https://mapzen.com/81d4d985-0c44-4a18-bf12-6c4b4dd762e0:42833:43)
    at blob:https://mapzen.com/81d4d985-0c44-4a18-bf12-6c4b4dd762e0:33915:40
    at <anonymous>
```

which traces back to:

```js
var outline_style = this.styles[draw.outline.style];
draw.outline.dash = draw.outline.dash !== undefined ? draw.outline.dash : outline_style.dash;
```

Looks like this doesn't handle the possibility that `draw.outline.style` might be undefined...

**EXPECTED RESULT:**

An error about a missing or malformed style would be cool.

**Comment by bcamper** [comment]
Addressed in 2b540a2131626e98a4b7863be869fb3f296eaf6b

**Comment by bcamper** [comment]
cc @hanbyul-here @rfriberg @burritojustice this line outline syntax issue will be addressed in Tangram v0.14.2 -- will log a warning and skip invalid outlines, rather than stopping all rendering.

**Comment by bcamper** [comment]
Released in [v014.2](https://github.com/tangrams/tangram/releases/tag/v0.14.2)
